domains/mapsyseo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "ShingSemicolon",
+    "email": "Shing.XD.0602@gmail.com"
+  },
+  "record": {
+    "A": ["81.16.29.155"]
```

Comment:
I cannot reach your website, 403 forbidden, Access to this resource on the server is denied!

Reply:
This IP can only be accessed through the Domain, to avoid DDoS attacks, this website is made with Hostinger.